### PR TITLE
Add token type if missing

### PR DESCRIPTION
**Pull request by miiu96**
- Reindex token type if case of missing in the `accountsesdt` index and the `tokens` index

### CODE REVIEW

**Review by dragos-rebegea — DISMISSED** [review]

File: data/account.go [hunk 1]
```diff
@@ -65,3 +66,13 @@ type AccountESDT struct {
 	IsSender        bool
 	IsNFTOperation  bool
 }
+
+// ResponseBulk defines the structure for response accounts ESDT bulk
```
Comment (bogdan-rosianu) [inline data/account.go:70]: `// ResponseBulk defines the generic structure for an Elasticsearch bulk request`
also, can be moved to a more generic file (not account.go) as we can use it in other places as well

File: go.mod [hunk 2]
```diff
@@ -8,4 +8,5 @@ require (
 	github.com/ElrondNetwork/elrond-vm-common v1.2.9
 	github.com/elastic/go-elasticsearch/v7 v7.12.0
 	github.com/stretchr/testify v1.7.0
+	github.com/tidwall/gjson v1.14.0
```
Comment (bogdan-rosianu) [inline go.mod:11]: :+1:

File: integrationtests/accountsESDTWithTokenType_test.go [hunk 3]
```diff
@@ -145,3 +150,137 @@ func TestIndexAccountESDTWithTokenType(t *testing.T) {
 	require.JSONEq(t, expectedAccountESDT, string(genericResponse.Docs[0].Source))
 
 }
+
+func TestIndexAccountESDTWithTokenTypeFirstShardAndAfterMetachain(t *testing.T) {
```
Comment (bogdan-rosianu) [inline integrationtests/accountsESDTWithTokenType_test.go:154]: `TestIndexAccountESDTWithTokenTypeShardFirstAndMetachainAfter`

File: process/accounts/serialize.go [hunk 4]
```diff
@@ -153,3 +153,26 @@ func prepareSerializedAccountBalanceHistory(
...
+
+// SerializeTypeForProvidedIDs will serialize the type for the provided ids
+func (ap *accountsProcessor) SerializeTypeForProvidedIDs(ids []string, tokenType string) ([]*bytes.Buffer, error) {
+	buffSlice := data.NewBufferSlice()
+
+	for _, id := range ids {
+		meta := []byte(fmt.Sprintf(`{ "update" : { "_id" : "%s", "_type" : "_doc" } }%s`, id, "\n"))
+
+		serializedDataStr := fmt.Sprintf(`{"scripted_upsert": true, "script": {`+
+			`"source": "if ( ctx.op == 'create' )  { ctx.op = 'noop' } else  { ctx._source.type = params.type }",`+
```
Comment (bogdan-rosianu) [inline process/accounts/serialize.go:165]: :medal_sports: :rocket:

File: integrationtests/accountsESDTWithTokenType_test.go (hunk 3 above)
Comment (miiu96) [reply integrationtests/accountsESDTWithTokenType_test.go:154]: renamed.

**Review by bogdan-rosianu — APPROVED** [review]

**Review by dragos-rebegea — APPROVED** [review]

**Pull request merged**